scripts/convert_stable_cascade.py

```diff
@@ -30,11 +30,16 @@
 parser.add_argument("--use_safetensors", action="store_true", help="Use SafeTensors for conversion")
 parser.add_argument("--save_org", type=str, default="diffusers", help="Hub organization to save the pipelines to")
 parser.add_argument("--push_to_hub", action="store_true", help="Push to hub")
+parser.add_argument("--variant", type=str, help="Set to bf16 to save bfloat16 weights")
```

Comment:
```suggestion
parser.add_argument("--variant", type=str, choices=["bf16", "fp16"] help="Set to bf16 to save bfloat16 weights.")
```

Both should be supported yeah?